src/services/__mocks__/keycloakService.ts

```diff
@@ -0,0 +1,9 @@
+class KeycloakServiceMock {
```

Comment:
More functions may need to be implemented to proceed with more tests for the login and logout operations.